MapleServer2/Servers/Login/LoginSession.cs

```diff
@@ -17,5 +17,9 @@ public class LoginSession : Session {
         public int GetToken() {
             return rng.Next();
         }
+
+        public override void EndSession()
+        {
```

Comment:
what is this doing ?

Reply:
I added an abstract method in `Session` so any class that implements it needs to have an override, even if we don't need it to do anything for now